src/main/java/stream/flarebot/flarebot/util/Filter.java

```diff
@@ -19,6 +22,12 @@ public FilterReply decide(ILoggingEvent event) {
                 return FilterReply.DENY;
             }
         }
+        String thread = event.getThreadName();
+        for (String prefix : threadBlacklist) {
+            if (thread.startsWith(prefix)) {
+                return FilterReply.DENY;
```

Comment:
Yeah that was a mistake due to me being tired. And again they are more often that not useful.